canonical-web-and-design/maas-ui
# refactor(ui): make tag form components reusable

## Done

- Make the tag form components reusable so they can be used in the edit tag from and machine list tag
form.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Visit the tag list and open the create tag form. It should work as before.
- Find a tag in the tag list that is applied to some nodes.
- There should be links to the nodes' lists with the tag filter applied.
- Click on the tag to go the tag details.
- There should be "applied to" node links as before.

## Fixes

Fixes: canonical-web-and-design/app-tribe#770.
